CODEOWNERS

```diff
@@ -11,6 +11,8 @@
 # compression extensions
 /*/extensions/compression/common/compressor @rojkov @junr03
 /*/extensions/compression/gzip/compressor @rojkov @junr03
+# alternate protocols chace extensions
```

Comment:
cache?

Reply:
Bah! check_spelling didn't catch that :) Done.